go.mod

```diff
@@ -10,11 +10,14 @@ replace (
 	sigs.k8s.io/controller-tools => github.com/christarazi/controller-tools v0.6.2
 )
 
+replace github.com/cilium/cilium => github.com/sayboras/cilium v1.11.0-rc2.0.20220502103038-3e6cc47e7e5f
```

Comment:
this is temp changes while waiting for https://github.com/cilium/cilium/pull/19662